post-processor/googlecompute-export/post-processor.go

```diff
@@ -183,6 +189,8 @@ func (p *PostProcessor) PostProcess(ctx context.Context, ui packersdk.Ui, artifa
 		SourceImageProjectId: []string{"compute-image-tools"},
 		Subnetwork:           p.config.Subnetwork,
 		Zone:                 p.config.Zone,
+		OmitExternalIP:       p.config.OmitExternalIP,
+		UseInternalIP:        p.config.UseInternalIP,
```

Comment:
My understanding is as follows
1. When UseInternalIP is true OmitExternalIP must be True. 
1. When OmitExternalIP is true UseInternalIP must be True. 
1. When UseInternalIP is false OmitExternalIP must be false. 
1.  OmitExternalIP can not be false when UseInternal is True
1. UseInternalIP can not be false when OmitExternalIP is true because of case 2. 

```suggestion
		OmitExternalIP:       p.config.UseInternalIP,
		UseInternalIP:        p.config.UseInternalIP,
```